src/scss/features/_subnav.scss

```diff
@@ -148,4 +148,12 @@
 		// don't download another icon...
 		transform: rotate(180deg);
 	}
+
+	.o-header__subnav-right-align {
```

Comment:
Looks good. Could I be pendantic and ask you to align the class name to our other modifiying classes?
If it is definitely just the link, something like `.o-header__subnav-link--right`?

Reply:
np done